### PR TITLE
RAC5278: remove nested parallel in post test groovy script

### PR DESCRIPTION
**Pull request by PengTian0**
- Add jobs/FunctionTest/PostTest.groovy to generate test branches of post test and trigger them in parallel

- Add jobs/FunctionTest/SourceBasedTest.groovy to generate test branches of function test and trigger them in parallel

- Add ```timestamps``` to groovy scripts  which are the entry point of pipelines (MasterCI aka)

- Remove ```timestamps``` from groovy scripts which are not the entry point of any pipeline

@panpan0000 @changev @anhou @iceiilin

### CODE REVIEW

**Comment by panpan0000** [comment]
test this please

**Comment by panpan0000** [comment]
Jenkins Test failed, please address the issue. "Manifest miss"

File: jobs/FunctionTest/SourceBasedTest.groovy [hunk 1]
```diff
@@ -0,0 +1,99 @@
+node{
+    deleteDir()
+    checkout scm
+    def shareMethod = load("jobs/ShareMethod.groovy")
+    def function_test = load("jobs/FunctionTest/FunctionTest.groovy")
+    def ALL_TESTS = function_test.getAllTests()
+    def used_resources = function_test.getUsedResources()
+    def test_branches = [:]
+
+    // ova post test
```
Comment (PengTian0) [inline jobs/FunctionTest/SourceBasedTest.groovy]: unused line

**Comment by PengTian0** [comment]
test this please

**Review by panpan0000 — APPROVED** [review]
+1 after Pair Programming Review.
although I think we can refactor those groovy with kind of polymorphism , but taken groovy doesn't support OOD and we will have to use some tricky way to achieve that , so will defer this work outside this PR.
I like this PR which can resolve the wrongly status report in pipeline stages.

**Review by anhou — APPROVED** [review]

**Pull request merged**